example/src/components/BlockchainActions.tsx

```diff
@@ -9,6 +9,9 @@ import type {
   FormattedRpcResponse,
 } from '../types/methods';
 import {
+  queryContract,
```

Comment:
Can I suggest to stick with naming of methods wagmi uses? https://wagmi.sh/core/getting-started

i.e. `getContract`, `readContract` and `writeContract`

there is also a lot of good stuff in their core docs to look at / get inspired by

Reply:
Great, i'll check their docs 💪